pkg/workloads/tf_api/api.py

```diff
@@ -379,6 +379,20 @@ def predict(deployment_name, api_name):
 
     return jsonify(response)
 
+def valid_model_dir(model_dir):
+    """
+    validates that model_dir has the expected directory tree, e.g:
+
+    {TF serving version number}/
+        saved_model.pb
+        variables/
+            variables.data-00000-of-00001
```

Comment:
is it always going to be variables.data-00000-of-00001?

Reply:
not sure, which is why i didn't check for this

Reply:
gotcha thanks